File: Angular-Project/WineSellingProject/src/app/Models/City.ts
import { ICity } from "../interfaces/ICity";

export class City implements ICity{
    Id: number;
    Country: string;
    PostalCode: string;
    CityName: string;

    constructor(Id: number, Country: string, PostalCode: string, CityName: string) {
        this.Id = Id;
        this.Country = Country;
        this.PostalCode = PostalCode;
        this.CityName = CityName;
    }
}

File: DAL/Models/City.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Models
{
    public class City
    {
      
        public int Id { get; set; }
        public string CityName { get; set; }
        public string? PostalCode { get; set; }
        public string Country { get; set; }

    }
}
